pages/landing/hipaa/template/pricing/index.html

```diff
@@ -0,0 +1,50 @@
+---
+layout: landing
+type: landing
+slug: index-page
+
+title: Pricing
+excerpt: Configure the plan just right for your team.
+permalink: /hipaa-cloud-compliance/pricing/
+
+extra_nav_links:
+  - name: Gruntwork HIPAA
+    url: /hipaa-cloud-compliance
+
+pricing:
+  - title: Gruntwork HIPAA Early Access
+    description: Early access subscribers will receive a CIS staging environment when invited<br>
+      and a fully HIPAA-compliant production infrastructure when it's ready.
+    price: 3060
+    strike_price: 4080
+    flat_fee: 7425
+    strike_flat_fee: 9900
+    image: /assets/img/logos-compliance/hipaa.png
+    image_styles: "max-width: 130px"
+    timeline:
+      - name: Become an early backer
+        description: One description
+      - name: Receive CIS
+        description: Two description
+      - name: Wait while we develop and improve
+        description: Three description
+      - name: Receive HIPAA
+        description: Four description
+      - name: Audit with Gruntwork
+        description: Five description
+    features:
```

Comment:
We need to flush out that you get access to the IaC Lib, Ref Arch, support, training, audit assistance, stage vs prod env, etc.

Reply:
Addressed in 5adb11e4

Reply:
This is now more clear thanks to Pete's latest updates.